extensions/amp-ooyala-player/0.1/validator-amp-ooyala-player.protoascii

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-ooyala-player
+  tag_name: "SCRIPT"
+  spec_name: "amp-ooyala-player extension .js script"
+  mandatory_parent: "HEAD"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-ooyala-player"
+    dispatch_key: true
+  }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-ooyala-player-(latest|0\\.1)\\.js"
+  }
+  attrs: {
+    name: "type"
+    value: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-ooyala-player.html"
+}
+tags: {  # <amp-ooyala-player>
+  tag_name: "AMP-OOYALA-PLAYER"
+  also_requires_tag: "amp-ooyala-player extension .js script"
+  attrs: {
+    name: "data-embedcode"
```

Comment:
would you like any attribute value constraints on these to help folks avoid typos? For example are data-embedcode or data-playerid only alphanumeric, only lower case, have a certain length, etc?

If not that's fine.